lib/src/impl_js/impl_js.dart

```diff
@@ -53,3 +55,12 @@ class _KeyPair<S, T> implements KeyPair<S, T> {
 
   _KeyPair({required this.privateKey, required this.publicKey});
 }
+
+const webCryptImpl = _WebCryptoImpl();
```

Comment:
avoid leaking the type, not that it really matters, it's just prettier in my head :rofl: 
```suggestion
const WebCryptoImpl webCryptImpl = _WebCryptoImpl();
```